ledger/src/blockstore_db.rs

```diff
@@ -117,6 +117,12 @@ pub enum IteratorMode<Index> {
     From(Index, IteratorDirection),
 }
 
+// When adding a new column ...
+// - Add struct below and implement `Column` and `ColumnName` traits
+// - Add descriptor in Rocks::open() and name in Rocks::columns()
+// - Account for column in both `run_purge_with_stats()` and
+//   `compact_storage()` in ledger/src/blockstore/blockstore_purge.rs !!
+// - Account for column in `analyze_storage()` in ledger-tool/src/main.rs
```

Comment:
Use `///` for function comment block as we previously discussed :p.

Reply:
Yeah, agreed that it is more internal guidelines than public facing docs that somebody would be interested; I'll throw it inside the the `columns` mod tho!